server/src/lib/model/db/migrations/20221209022521-enable-dataset-downloads.ts

```diff
@@ -0,0 +1,21 @@
+export const up = async function (db: any): Promise<any> {
+  await db.runSql(`
+    UPDATE datasets
+    SET is_deprecated = 0
+    WHERE release_dir in (
+      'cv-corpus-1',
+      'cv-corpus-2', 
+      'cv-corpus-3',
+      'cv-corpus-4-2019-12-10',
+      'cv-corpus-5.1-2020-06-22',
+      'cv-corpus-6.1-2020-12-11',   
+      'cv-corpus-7.0-2021-07-21',
+      'cv-corpus-8.0-2022-01-19',
+      'cv-corpus-9.0-2022-04-27'
+      )
+  `);
+};
+
+export const down = function (): Promise<any> {
+  return null;
```

Comment:
What about the down migration where we set `is_deprecated` back to 1? Is that needed?

Reply:
Generally since this is a "content" update, not a schema update,  I didnt create an equivalent `down` as the values are both correct (true or false)